okhttp/src/test/java/io/grpc/okhttp/OkHttpClientTransportTest.java

```diff
@@ -181,16 +181,6 @@ public long read() {
     }
   }
 
-  /** Final test checks and clean up. */
-  @After
-  public void tearDown() throws Exception {
```

Comment:
Instead of removing this and requiring all the tests but one call the shutdownAndVerify() (where it may be viewed as a bug in the test), you could add a `clientTransport != null` check instead.